low/yanff_ring.h

```diff
@@ -17,7 +17,7 @@ yanff_ring_create(const char *name, unsigned count, int socket_id, unsigned flag
 	r = malloc(sizeof(struct yanff_ring));
 
 	r->DPDK_ring = rte_ring_create(name, count, socket_id, flags);
-	r->internal_DPDK_ring = r->DPDK_ring->ring;
-	r->offset = sizeof(r->DPDK_ring->ring[0]);
+	r->internal_DPDK_ring = &(r->DPDK_ring)[1];
```

Comment:
Please clarify why such change. I see in dpdk 17.08 rte_ring has no void* ring field, but still the new value is not obvious

Reply:
It is beginning of ring. Ring data is placed after rte_ring structure. So if we imagine that rte_ring is an array, so ring[1] is just after rte_ring structure itself. So it points just to beginning of ring data. I will add a comment.